sklearn/cluster/tests/test_k_means.py

```diff
@@ -190,6 +191,7 @@ def test_kmeans_convergence(algorithm):
     assert km.n_iter_ < max_iter
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")  # TODO(1.3): Remove
```

Comment:
We can use a `pytestmark` at the top of this file to ignore the new warning instead of catching it in call.